main/res/configs/playerSpawn.json

```diff
@@ -1,7 +1,7 @@
 {
   player: {
     ship: {
-      hull: imperialSmall,
+      hull: imperialTiny,
```

Comment:
Just for now I'd like to keep the default as imperialSmall, since it is used in the tutorial, the videos, etc. Down the road when we make it easier to add ships and maybe define factions and such we could switch around the defaults some :-)